package.json

```diff
@@ -54,6 +52,13 @@
     ]
   },
   "devDependencies": {
-    "cross-env": "^6.0.3"
+    "cross-env": "^6.0.3",
```

Comment:
might want to add a script for ppl to run `npm run lint` that runs sth like `eslint . --ext .js .tsx'`